powerquery-docs/samples/trippin/3-navtables/readme.md

```diff
@@ -93,34 +93,9 @@ TripPin.Contents("https://services.odata.org/v4/TripPinService/")
 
 ## Creating a navigation table
 
-You'll use the handy [Table.ToNavigationTable](../../../handling-navigation-tables.md#tabletonavigationtable) function to format your static table into something that Power Query will recognize as a navigation table.
+You'll use the handy [Table.ToNavigationTable](../../../helper-functions#tabletonavigationtable) function to format your static table into something that Power Query will recognize as a navigation table. Since this function is not part of Power Query's standard library, you'll need to copy its source code into your .pq file.
```

Comment:
Probably should be a .md file link
helper-functions.md#tabletonavigationtable

Reply:
Good catch. Thanks!